app/theme/client/livechat.css

```diff
@@ -0,0 +1,398 @@
+.inner-addon {  
+    position: relative;
+    left: 24px;
+    box-sizing: border-box;
+    width: 345px;
+    height: 40px;
+    border: 2px solid #cbced1;
+    border-radius: 2px;
+    background: #ffffff;
+}
+
+.inner-addon .glyphicon {
+    position: absolute;
+    bottom: 0.5%;
+    color: #2f343d;
+    padding: 10px;
+    pointer-events: none;	
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5378317)